code/modules/mob/living/living_defense.dm

```diff
@@ -204,6 +204,7 @@
 			hitpush = FALSE
 			skipcatch = TRUE
 			blocked = TRUE
+			throwingdatum.blocked = TRUE
```

Comment:
This shouldn't be necessary - `hitby` is supposed to `return TRUE` if it was blocked, so as I mentioned before, it should be returning `TRUE` here.